docs/whats-new.md

```diff
@@ -397,7 +397,7 @@ The following table compares the performance between CPU and GPU aggregations us
 
 ### Use react-map-gl Components with DeckGL
 
-For React users, it is now easy to use [react-map-gl](http://uber.github.io/react-map-gl/#/Examples/markers-popups) components with DeckGL, including DOM-based [marker](https://uber.github.io/react-map-gl/#/Documentation/api-reference/marker), [popup](https://uber.github.io/react-map-gl/#/Documentation/api-reference/popup), [navigation control](https://uber.github.io/react-map-gl/#/Documentation/api-reference/navigation-control), [fullscreen control](https://uber.github.io/react-map-gl/#/Documentation/api-reference/fullscreen-control) and [geolocate control](https://uber.github.io/react-map-gl/#/Documentation/api-reference/geolocate-control). This can be done by supplying the `ContextProvider` prop on `DeckGL`:
+For React users, it is now easy to use [react-map-gl](http://visgl.github.io/react-map-gl/#/Examples/markers-popups) components with DeckGL, including DOM-based [marker](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/marker), [popup](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/popup), [navigation control](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/navigation-control), [fullscreen control](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/fullscreen-control) and [geolocate control](https://visgl.github.io/react-map-gl/#/Documentation/api-reference/geolocate-control). This can be done by supplying the `ContextProvider` prop on `DeckGL`:
```

Comment:
These links are outdated

Reply:
Looks these links refer to old react-map-gl website demos, looks like lot of things changed, not sure what the correct links are now, I don't have the context here.

Reply:
For the what's new in luma.gl, if the links were no longer valid (e.g. old lesson examples), I just removed them.